app/lib/api_clients/twilio_verify_client.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rubygems'
+require 'twilio-ruby'
+
+# Help: https://www.twilio.com/docs/verify/quickstarts/totp
+class TwilioVerifyClient
+  TWILIO_ACCOUNT_SID = ENV['TWILIO_ACCOUNT_SID']
+  TWILIO_AUTH_TOKEN = ENV['TWILIO_AUTH_TOKEN']
+  TWILIO_SMS_SERVICE = ENV['TWILIO_SMS_SERVICE']
```

Comment:
We have historically used placeholder values & payloads, I think @colinxfleming's recent work with pledge generation is a solid model - see below for how it has default values set via `ENV.fetch()`.

https://github.com/DARIAEngineering/dcaf_case_management/blob/4e0dc7d112cb4c80ee539d85018b8b8d9bc1c58e/app/controllers/patients_controller.rb#L58-L63

Since the tests are built on mocks of the API (🎉), I think we just need to set these to values that will allow the class to initialize cleanly.

(review still very WIP - lemme know if this makes sense or I've missed anything critical)

Reply:
Makes sense, thanks! I'll add a default value with a placeholder for each of these so it'll initialize for tests at least. Then the API call will still fail if the actual ENV variables aren't present, but I suppose that's not a bad thing. I can add a comment for developers to find the values in the Twilio console if they need to work on MFA.

Reply:
Actually I tried to replicate locally and it looks like the test issue might be unrelated to this. I'll keep debugging then come back to what changes here are necessary.

Reply:
Fixed now! The errors were due to the autoloader not being able to find the `TwilioVerifyClient` class in the `app/lib/api_clients` directory (it was expecting `ApiClients::TwilioVerifyClient`). I opted to remove the `/api_clients` directory and put the file directly into `app/lib` since it was the only API client we had. Seemed better than having to write `ApiClients:TwilioVerifyClient` everywhere.

I didn't change to using `ENV.fetch` since tests pass as-is but I'm happy to go back and do that if you'd prefer.